blog/2023-07-27-introducing-substratus.md

```diff
@@ -36,6 +36,11 @@ There's got to be a better way.
 
 ## Why Substratus?
 
+**Faster time to market of custom ML models**: Load a base model, finetune it
+with your own dataset and serve it without having to write any code. Substratus
+provides out of the box container images to make zero code finetuning a reality.
+Need more flexibility? Simply clone the official image and adapt it to your needs.
+
```

Comment:
This is a good point of emphasis. A note to self: re-org this list to put what's most compelling at the very top. Avoiding vendor lock-in is decidedly not our most important selling point.